cli/src/main.rs

```diff
@@ -541,7 +557,15 @@ fn parse_arguments() -> ArgMatches<'static> {
                 .add_input_model_arg()
                 .add_input_model_type_arg()
                 .add_formula_arg()
-                .add_formula_type_arg(),
+                .add_formula_type_arg().arg(
+                Arg::with_name("quiet")
+                    .short("q")
+                    .takes_value(false)
+                    .long("quiet")
+                    .help(
+                        "Suppress stdout and only return exitcode 42 for true result or 43 for false",
+                    ),
```

Comment:
Since this is now applies to two subcommands it should be moved to `cli::args`

Reply:
Fixed in e6e2663